zuul.d/adoption.yaml

```
@@ -95,9 +95,15 @@
                 ip: 172.20.0.100
                 config_nm: false
 
+# TODO(marios): remove this once we finish rename OSPRH-8452
 - job:
     name: cifmw-data-plane-adoption-osp-17-to-extracted-crc
-    parent: data-plane-adoption-osp-17-to-extracted-crc
+    parent: adoption-standalone-to-crc-ceph-provider
+
+- job:
+    name: adoption-standalone-to-crc-ceph-provider
+    description: Standalone source OSP 17.1 node adopted to crc RHOSO 18. Has ceph, not TLS. Uses a content provider.
```

We can use pipe here and make the description multi line like this. What do you say?
```suggestion
    description: |
         Standalone source OSP 17.1 node adopted to crc RHOSO 18. 
         Has ceph, not TLS. Uses a content provider.
```

---

sure